README.md

```diff
@@ -1113,6 +1114,11 @@ _Frameworks that handle the communication between the layers of a web applicatio
 - [Takes](https://github.com/yegor256/takes) - Opinionated web framework which is built around the concepts of True Object-Oriented Programming and immutability.
 - [Vaadin](https://vaadin.com) - Event-driven framework that uses standard web components. Server-side architecture with Ajax on the client side.
 
+### Workflow Orchestration Engines
+
+- [Cadence](https://cadenceworkflow.io) - Stateful code platform from Uber.
+- [Temporal](https://temporal.io) - Microservice orchestration platform, forked from Cadence but gRPC based.
+
```

Comment:
The descriptions and the section naming are the tough part would be glad if somebody suggested something better